modules/win-autoscale/watchmaker-win-autoscale.cfn.json

```diff
@@ -660,6 +679,43 @@
                                     ]
                                 },
                                 "waitAfterCompletion": "0"
+                            },
+                            "11-cloudwatch-agent-install": {
+                                "command": {
+                                    "Fn::If": [
```

Comment:
Rather than having the `if` statement here, I think it is cleaner to handle it through the config-set. Example:

* https://github.com/plus3it/terraform-aws-watchmaker/blob/fc23b21cd9b44f2303561549f78ae92339fee588/modules/win-autoscale/watchmaker-win-autoscale.cfn.json#L478-L486